core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/OperatorUtils.java

```diff
@@ -171,4 +171,69 @@ public String toString() {
    */
   public static final BiPredicate<ExprValue, ExprValue> COMPARE_WITH_NULL_OR_MISSING =
       (left, right) -> left.isMissing() || right.isMissing() || left.isNull() || right.isNull();
+
+  /**
+   * Wildcard pattern matcher util.
+   * Percent (%) character for wildcard,
+   * Underscore (_) character for a single character match.
+   * @param pattern string pattern to match.
+   * @return if text matches pattern returns true; else return false.
+   */
+  public static boolean matches(String text, String pattern) {
+    return Pattern.compile(patternToRegex(pattern)).matcher(text).matches();
```

Comment:
I checked a few methods in JDK, such as `Pattern.matches`. I think it's more intuitive to put pattern as first argument for this util method as well as like operator argument?

Reply:
Good point! Will change the argument order in the util method to follow the JDK custom. As for the like operator, I think it would be better to keep consistent with the PPL specification, which is text like pattern.

Reply:
I see. I wan't aware of that. Thanks!